src/main/java/com/bfa/exec/Main.java

```diff
@@ -0,0 +1,17 @@
+package com.bfa.exec;
+
+import com.bfa.core.BrowserType;
+import com.bfa.core.page.GooglePage;
+
+/**
+ * Created by ninhdoan on 12/24/16.
+ */
+public class Main {
+
+   public static void main(String[] args) {
+      GooglePage.openGoogle(BrowserType.FIREFOX)
+            .search("just a test")
```

Comment:
Please update to search something more meaningful, like ```Hello World```, or ```Selenium 3 WebDriver```. The one ```just a test``` is quite funny to be released.